connectd/connectd.c

```diff
@@ -151,6 +151,9 @@ struct daemon {
 
 	/* Allow to define the default behavior of tot services calls*/
 	bool use_v3_autotor;
+
+	/* What network are we running on ? */
+	const struct chainparams *chainparams;
```

Comment:
In other daemons we have a global instead, which might cause some discrepancies if we reuse code meant for other daemons in `connectd`. We should probably make this a global as well just in case.

Reply:
I also tried using the constant in `common/utils.h`, hmm reading other daemons maybe I could just populate this one in `fromwire_connectctl_init` insead.. Will try